docs/Installation/Installation.md

```diff
@@ -6,19 +6,121 @@ See LICENSE.txt for details.
+precompiled executables, which are published on GitHub:
+
+- Releases with precompiled executables:
+  https://github.com/sxs-collaboration/spectre/releases
+
+We only publish a limited set of precompiled executables that are useful as
+stand-alone tools, such as the CCE executables (see \ref tutorial_cce).
+
+### Quick-start guide for code development with Docker and Visual Studio Code
+
+If you're new to writing code for SpECTRE and would like to jump right into a
+working development environment, a good place to start is our
+\subpage dev_guide_quick_start_docker_vscode.
 
-### Install a specific version with Spack
+### Quick-start installation {#quick_start_install}
 
-You can install SpECTRE with the [Spack](https://github.com/spack/spack) package
-manager:
+The easiest way of installing SpECTRE natively on a new machine is this:
+
+1. Collect dependencies. You need a C++ compiler (GCC or Clang), CMake,
+   BLAS/LAPACK, Boost, GSL, HDF5, and Python installed. For details on these
+   required dependencies see \ref build_dependencies. On many computing clusters
+   they are available as modules. On personal machines you can install them with
+   a package manager.
+
+2. Clone the SpECTRE repository:
+
+  ```sh
+  git clone git@github.com:sxs-collaboration/spectre.git
+  export SPECTRE_HOME=$PWD/spectre
+  ```
+
+3. Install Charm++:
+
+  ```sh
+  git clone https://github.com/UIUC-PPL/charm
+  cd charm
+  git checkout v7.0.0
+  git apply $SPECTRE_HOME/support/Charm/v7.0.0.patch
+  ./build charm++ <version> --with-production --build-shared
+  export CHARM_ROOT=$PWD/<version>
+  ```
+
+  Choose the `<version>` from [this list in the Charm++
+  documentation](https://github.com/charmplusplus/charm?tab=readme-ov-file#how-to-choose-a-version).
+  For example, choose `multicore-linux-x86_64` on a Linux laptop,
+  `multicore-darwin-arm8` on an Apple Silicon laptop, and `mpi-linux-x86_64` on
+  a standard computing cluster (you will also need MPI for this). See
+  \ref building-charm for details.
+
+4. Configure and build SpECTRE:
+
+  ```sh
+  cd $SPECTRE_HOME
+  mkdir build
+  cd build
+  cmake \
+    -D CMAKE_C_COMPILER=<clang or gcc> \
+    -D CMAKE_CXX_COMPILER=<clang++ or g++> \
+    -D CMAKE_Fortran_COMPILER=gfortran \
+    -D CMAKE_BUILD_TYPE=<Debug or Release> \
+    -D CHARM_ROOT=$CHARM_ROOT \
+    -D SPECTRE_FETCH_MISSING_DEPS=ON \
+    -D MEMORY_ALLOCATOR=SYSTEM \
+    $SPECTRE_HOME
+  ```
+
+  See \ref building-spectre for details and \ref common_cmake_flags for a list
+  of possible configuration options. For example, set `-D ENABLE_OPENMP=ON`
+  to enable OpenMP-parallelization for the exporter library.
+
+  Now you can compile executables (again, see \ref building-spectre for
+  details). For example:
+
+  ```sh
+  make -j12 cli
+  make -j12 BundledExporter
+  ```
+
+### Installation with Spack
+
+You can also install SpECTRE with the [Spack](https://github.com/spack/spack)
+package manager:
```

Comment:
Do we want to warn people that this may not work for a number of reasons? (local env, spack upstream, etc.)